mobify/platform-scaffold
# Ensure Class is Self-Documenting

## Changes
- Rename `t-home__nearest-stores` to `t-product-details__nearest-stores`

## How to test-drive this PR
- Run `npm run dev`
- Preview [Merlin's Potions](https://preview.mobify.com/?url=https%3A%2F%2Fwww.merlinspotions.com%2F&site_folder=https%3A%2F%2Flocalhost%3A8443%2Floader.js&disabled=0&domain=&scope=1)
- Visit any PDP
- Ensure the "Nearest Store" elements still render as intended
